enhancements/operator-conditions.md

```diff
@@ -64,13 +64,18 @@ As an operator author, I want operators managed by OLM to be able to communicate
 
 ## Design Details
 
-### Condition CRD
+### status.operators.coreos.com CRD
 
-OLM will create a channel for operators to communicate complex conditions by introducing the `Condition` [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+OLM will create a channel for operators to communicate complex conditions by introducing the `Status` [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
 
-When an operator is installed, OLM will create a `Condition` CustomResource (CR) for the operator. The operator should only be concerned with updating the status of the `Condition` that it owns and will have its RBAC scoped so it may only GET, LIST, or UPDATE the `Condition` CR it is associated with.
+When an operator is installed, OLM will create a `Status` CustomResource (CR) for the operator. The operator's RBAC will be configured such that it is only able to get the `Status` resource it own and update the resources [status subresource](https://book-v1.book.kubebuilder.io/basics/status_subresource.html).
```

Comment:
IMO, `Status` is even more confusing that `Conditions` b/c of the status subresource. Perhaps its best for us to get away from overloaded terms, maybe `Probe` is more apt (slightly less overloaded).

Reply:
I don't believe that probe is appropriate given that the operator is basically communicating its status to OLM via this resource (I am purposely going to advocate that we avoid discussing the features in #23 that led to this name, but if we do add these features in the future we could do so with a spec.probe field).

Reply:
There was also concern that the `Conditions` name is just as overloaded as `Status` and limits the CRD from expanding to additional usecases outside of communicating state through the `status.conditions` array.